lib/oli_web/components/delivery/practice_activities/practice_assessments_table_model.ex

```diff
@@ -0,0 +1,163 @@
+defmodule OliWeb.Delivery.PracticeActivities.PracticeAssessmentsTableModel do
+  use Phoenix.Component
+
+  alias OliWeb.Common.Table.{ColumnSpec, SortableTableModel}
+  alias OliWeb.Common.FormatDateTime
+  alias Phoenix.LiveView.JS
+
+  def new(assessments, ctx, target) do
+    column_specs = [
+      %ColumnSpec{
+        name: :order,
+        label: "ORDER",
+        render_fn: &__MODULE__.render_order_column/3,
+        th_class: "pl-10"
+      },
+      %ColumnSpec{
+        name: :title,
+        label: "ASSESSMENT",
+        render_fn: &__MODULE__.render_assessment_column/3,
+        th_class: "pl-10"
+      },
+      %ColumnSpec{
```

Comment:
Practice pages will usually never have a Due Date.  This column should be removed.